scripts/e2e.sh

```diff
@@ -0,0 +1,71 @@
+MY_PATH="`dirname \"$0\"`"              # relative
```

Comment:
@inodb @pvannierop this is script i have written to hopefully make booting up localdb mostly single command.

Reply:
thumb sup